templates/aframe-model-template/app/assets/index.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Default scene! • Made with A-Frame</title>
+    <link rel="stylesheet" href="css/app.css">
+    <script src="js/vendor.js"></script>
+    <script src="js/app.js"></script>
+    <script>
+    AFRAME.registerComponent('teleport', {
```

Comment:
Not fussy about it.

initialize.js is a bit confusing to me since I don't know what it's purpose is.  Stuffing away into it's own component is fine, but as you say, hides how it works.  Might be the best approach since it's a bit of a pattern already with the A-Frame community.